android-json-form-wizard/src/main/java/com/vijay/jsonwizard/fragments/JsonFormFragment.java

```diff
@@ -106,6 +137,48 @@ private void setupToolbarBackButton() {
 
     }
 
+    protected void initializeBottomNavigation(JSONObject step, View rootView) {
+        if (step.has(PREVIOUS)) {
+            previousButton.setVisibility(View.VISIBLE);
+            if (step.has(PREVIOUS_LABEL)) {
+                previousButton.setText(step.optString(PREVIOUS_LABEL));
+            }
+        }
+
+        if (step.has(NEXT)) {
+            nextButton.setVisibility(View.VISIBLE);
+            if (step.has(NEXT_LABEL)) {
+                nextButton.setText(step.optString(NEXT_LABEL));
+            }
+        }  else if (step.optString(NEXT_TYPE).equalsIgnoreCase(SUBMIT)) {
```

Comment:
Could you render Save button even when one has not specified next tags. This should be hocked to Save form for standardization with the current toolbar buttons.